add current mailinglist/wiki topics

sarumpaet opened the pull request:
This uses a small PHP script (that will be run on the server in
its hourly cronjob) to scrape recent topics from the mailinglist
and wiki server.

Code's not beautiful but not the nightmare to maintain that a Cyrax plugin would be; it's not as brittle as a JavaScript client solution would be; and it's more hackable than an RSS-based solution would be (it's already doing some filtering that would be difficult with RSS).

I'll be merging this soon.

booo commented on www/wiki.html:
```diff
@@ -20,5 +20,9 @@
 Mach mit!
 </p>
+<p>
+<b>Die letzten Berlin-spezifischen <a href="https://wiki.freifunk.net/index.php?title=Spezial:Letzte_%C3%84nderungen&amp;days=30&amp;from=&amp;limit=500" class="externalLink">Änderungen im Wiki</a> gingen um die Themen...</b>
```
Der Satz ist komisch...

booo commented:
What happens if the script fails? Is it possible to inject e.g. javascript code with the mail topic? Should we add some escaping?

Python code for the script would be more consistent with the rest of the code base (config, ca).

sarumpaet commented:
I added explicit exit if the script's fetch fails. The cronjob will not update the website then. - The script works on raw HTML, not DOM, so everything is escaped already. Still, to be sure, I added tag/"<" escaping.
